conda/gateways/logging.py

```diff
@@ -203,25 +204,39 @@ def initialize_std_loggers():
 
 
 def initialize_root_logger(level=ERROR):
-    attach_stderr_handler(level)
+    attach_stderr_handler(level=level)
 
 
 def set_conda_log_level(level=WARN):
-    conda_logger = getLogger('conda')
-    conda_logger.setLevel(level)
-    conda_logger.propagate = True  # let root logger's handler format/output message
+    conda_logger = getLogger("conda")
+    conda_logger.setLevel(logging.NOTSET)
```

Comment:
This and the equivalent change in `conda.common.io` causes `log.isEnabledFor(DEBUG)`-guarded code to be run when not intended.
xref: https://github.com/conda/conda/issues/13541

Reply:
Nice find!

Reply:
Oopsie 😬